op-batcher/metrics/metrics.go

```diff
@@ -326,8 +341,14 @@ func estimateBatchSize(block *types.Block) uint64 {
 		if tx.IsDepositTx() {
 			continue
 		}
-		// Add 2 for the overhead of encoding the tx bytes in a RLP list
```

Comment:
maybe retain this comment in the `else` clause, or just remove the else clause altogether

Reply:
the `else` clause is not compatible with the other anyway, given one is compressed and one is not

Reply:
removed